CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support! Please have a look at the [issues page on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) and check if anybody already raised an issue with the same topic.
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+If this is the case, please subscribe to the issue and explain your proposal.
+
+If that's not the case, please open a [new issue](https://github.com/jamulussoftware/jamuluswebsite/issues/new/choose), select which type of issue it is and describe your issue.
+
+If you want to open a Pull Request (after having described and discussed your proposal), have a look at the [Contribution page on the website](https://jamulus.io/wiki/Contribution). Keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is allright) with other changes and upload multiple changes in one bulk to jamulus.io.
```

Comment:
Do we really want to link this page? I don't feel comfortable by doing so. In fact, we should link this file on the mentioned site not the other way round, I think

Reply:
> we should link this file on the mentioned site not the other way round, I think

I think too.

Reply:
Should be done

Reply:
What does "please subscribe to the issue" mean?

Reply:
![Subscribe to issue](https://user-images.githubusercontent.com/20726856/101404908-90006080-38d7-11eb-971a-93dd6c3c5b3c.png)

It's a GitHub feature

Reply:
Yes, but why do we ask them to do this? It seems oddly specific.

Reply:
@gegeweb?

I think it's since a user will then be able to receive updates on this issue. If you want to remove it, just do so.

Reply:
> I think it's since a user will then be able to receive updates on this issue. If you want to remove it, just do so.

Sorry, I wasn't online last night.
I'll let you decide (it seems to be updated), it's not necessarily useful to keep it.